ScoutSuite/providers/azure/facade/graphrbac.py

```diff
@@ -8,8 +8,8 @@
 class GraphRBACFacade:
     def __init__(self, graphrbac_credentials, credentials, tenant_id, subscription_id):
         self._subscription_id = subscription_id
-        self._client = GraphRbacManagementClient(graphrbac_credentials, tenant_id=tenant_id)
-        self._authorization_client = AuthorizationManagementClient(credentials, subscription_id=subscription_id)
+        self._client = GraphRbacManagementClient(graphrbac_credentials, tenant_id=tenant_id, base_url=graphrbac_credentials.cloud_environment.endpoints.resource_manager)
```

Comment:
This is required specifically for other environments? I'm asking as it isn't required in the default cloud environment...

Reply:
Passing the base_url ensures that the proper resource_manager endpoint is used, one of the breakpoints when using Azure gov.